linux_os/guide/services/sssd/sssd_offline_cred_expiration/bash/shared.sh

```diff
@@ -4,4 +4,10 @@
 # complexity = low
 # disruption = medium
 
+# sssd configuration files must be created with 600 permissions
```

Comment:
It isn't the best comment because it doesn't explain why.

Reply:
Expanded a bit the comment. I believe it should now be resolved.